src/test/java/org/cactoos/number/SumOfTest.java

```diff
@@ -125,7 +125,7 @@ void withIterableOfInts() {
     void overflowIntFromLongValues() {
         MatcherAssert.assertThat(
             new SumOf(2_147_483_647L + 1L << 1, 10L).intValue(),
-            new IsEqual<>(2_147_483_647)
+            new IsEqual<>(10)
```

Comment:
@svendiedrichsen I think you originally wrote this test, and I wasn't sure exactly what it was supposed to verify or test, but after changing the way SumOf is implemented, now it gives this result. Do you think this is correct and if not, could you help me understand why? thx!

Reply:
@victornoel I have initially written this test to assure that long overflows during summation don't affect the result. Original commit here: https://github.com/yegor256/cactoos/commit/cb97bf0c302c7d63413e3ca27ef5be812ec3a072#diff-68e784b552b842bf92ef1a42843f896862c82e87df36dcd69d8cd2561b1ad3b4

Reply:
@svendiedrichsen thank you for your answer. I think the commit you linked to did not change: I was referring to the test this comment here is attached to. I understand the general why, I had seen the commits and messages, but for this particular test, I don't exactly understand why the previous behaviour is correct, and why this new one would not. If you have some insight about it, it would be really helpful :)

Reply:
@victornoel I frankly can't remember anymore. Sorry.

Reply:
@svendiedrichsen ok, thanks :) I should find a satisfying solution I think, cheers